Cython/Compiler/PyrexTypes.py

```diff
@@ -256,6 +258,8 @@ class PyrexType(BaseType):
     is_struct_or_union = 0
     is_cpp_class = 0
     is_optional_cpp_class = 0
+    is_python_type_constructor = 0
+    is_special_python_type_constructor = 0
```

Comment:
We should stop writing `0/1` for `False/True`, and also covert old code when we touch it. Python has had a boolean type [since Python 2.3](https://www.python.org/dev/peps/pep-0285/). It's about time to use it.
```suggestion
    is_python_type_constructor = False
    is_special_python_type_constructor = False
```